### PR TITLE
tests: disable reporting to mongo

### PR DESCRIPTION
**Pull request by valentindavid**

### CODE REVIEW

**Review by bboozzoo — APPROVED** [review]
Thanks

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -888,23 +888,6 @@ jobs:
           spread -discard -reuse-pid="$(echo "$r" | grep -o -E '[0-9]+')";
         done
 
-    - name: report spread errors
```
Comment (pedronis) [inline .github/workflows/test.yaml:891]: how can we make sure not to forget to leave this out when we merge back?

Comment (bboozzoo) [reply .github/workflows/test.yaml:891]: hm I suppose leaving `# TODO placeholder for spread error reporting` should cause a conflict

Comment (valentindavid) [reply .github/workflows/test.yaml:891]: There will be a conflict. Because this is modified in master.

**Comment by bboozzoo** [comment]
This will be superseded with https://github.com/canonical/snapd/pull/14637

**Pull request closed without merging**